# Copyright 2023, Advanced Micro Devices, Inc
# SPDX-License-Identifier: GPL-2.0 WITH Linux-syscall-note
# Downloaded from https://support.xilinx.com/s/article/000035732?language=en_US
From cebd0f4c68a63a279814ce443a90ceb4f2d5353c Mon Sep 17 00:00:00 2001
From: Rajesh Gugulothu <rajesh.gugulothu@amd.com>
Date: Fri, 6 Oct 2023 18:29:17 +0530
Subject: [LINUX PATCH 5/5] drm: xlnx: hdmi: Force the driver to work only in
 TMDS mode

Currently driver does not support FRL mode so force the driver to
work in TMDS mode only.

Signed-off-by: Rajesh Gugulothu <rajesh.gugulothu@amd.com>
---
 drivers/gpu/drm/xlnx/xlnx_hdmi.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/gpu/drm/xlnx/xlnx_hdmi.c b/drivers/gpu/drm/xlnx/xlnx_hdmi.c
index a5b5a2040d19..37480e963d26 100644
--- a/drivers/gpu/drm/xlnx/xlnx_hdmi.c
+++ b/drivers/gpu/drm/xlnx/xlnx_hdmi.c
@@ -2820,6 +2820,7 @@ xlnx_hdmi_set_frl_tmds_mode(struct drm_connector *connector)
 	} else {
 		hdmi->stream.is_frl = 0;
 	}
+		hdmi->stream.is_frl = 0;
 }
 
 static int xlnx_hdmi_connector_get_modes(struct drm_connector *connector)
-- 
2.17.1

